### DISCUSSION TITLE
Explaining the Structure of Examples

### DISCUSSION THREAD

**Discussion by Baltmann01** [r-devel/cran-cookbook#34] [Ideas]
I started drafting the next recipe: Example structure and wrappers (\dontrun{}, \donttest{}, etc...). This is a fairly long section, at least compared to what we added already, as I would explain every wrapper and when to use it. 
Right now, I started typing it in the Documentation Issues. 
Should we rather add an overview under General Issues and just reference the bits and pieces for the actual recipe?
Or is it better to keep everything in the same place, in the Documentation Issues page?

**Comment by jasdumas** [comment]
As long as each structure/wrapper has an anchor tag to enable linking out from the header, I'm not to concerned about length. You could also make an overall mention of this recipe type in the `General` issue section

**Reply by Baltmann01** [reply]
Sounds good! I will write about the general structure in the General Issues and then talk aboit the common issues seperatly in the recipes.
